README.md

```diff
@@ -12,7 +12,7 @@ Open a command console, enter your project directory and execute the
 following command to download the latest release candidate for this bundle:
```

Comment:
should be updated to latest stable version if we want to do this change.

Reply:
Done https://github.com/thephpleague/tactician-bundle/commit/3e3af2ca6a98fefd6ce6d9f02a3df8fb38b71d21#diff-04c6e90faac2675aa89e2176d2eec7d8